packages/go-ui-storybook/src/stories/Table.stories.tsx

```diff
@@ -302,5 +310,6 @@ export const WithResizableColumn: Story = {
         keySelector,
         caption: 'You can utilize the header column to adjust the width of each column.',
         resizableColumn: true,
+        headerRowClassName: 'table-header-row',
```

Comment:
This is not correct. The header row color should be updated in the Table component itself.